github-action.py

```diff
@@ -75,6 +76,19 @@
 log = QubesBuilderLogger
 
 
+def get_log_file_from_qubesbuilder_buildlog(stdout, logger=None):
+    lines = str(stdout).splitlines()
+    if not stdout or not lines:
+        return
```

Comment:
The problematic case (in the reported issue) will hit this - so, it should also result in an error message logged as below, no?

Reply:
yes good point.